base/model/src/main/java/org/eclipse/ditto/base/model/entity/id/RegexPatterns.java

```diff
@@ -61,18 +61,24 @@
     public static final String ALLOWED_CHARACTERS_IN_NAME = NO_CONTROL_CHARS_NO_SLASHES;
 
     /**
-     * Adds the dot to allowed characters. Its defined as separate constant because namespaces are not allowed to start
-     * with dots.
+     * Adds additional characters that are allowed inside a label. It's defined as separate constant because namespaces
+     * are not allowed to start with those characters.
      */
-    public static final String ALLOWED_NAMESPACE_CHARACTERS_INCLUDING_DOT =
-            "\\." + ALLOWED_NAMESPACE_CHARACTERS_REGEX;
+    public static final String ALLOWED_NAMESPACE_CHARACTERS_REGEX_INNER =
+            "([.\\-])" + ALLOWED_NAMESPACE_CHARACTERS_REGEX;
+
+    /**
+     * @deprecated Use {@link #ALLOWED_NAMESPACE_CHARACTERS_REGEX_INNER}. The name of the variable actually described
+     * the content, which has now changed. The name new describes the purpose.
+     */
+    public static final String ALLOWED_NAMESPACE_CHARACTERS_INCLUDING_DOT = ALLOWED_NAMESPACE_CHARACTERS_REGEX_INNER;
```

Comment:
please - in addition to the `@deprecated` javadoc (well done :)) - please add annotation `@Deprecated` to the constant as well